packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -272,19 +272,11 @@ The factor profile settings object contains the settings for the particular fact
 
 #### Password Factor Profile Settings Object
 
-TODO
-
-| Property | Type | Description |
-| -------- | ---- | ----------- |
-| `.`      | .    | .           |
+This object currently does not contain any properties.
```

Comment:
then why are we mentioning it? same applies below

Reply:
They're mentioned because they will show up in the factor profile json as empty objects.  That said, we're not discussing the factor profile json, so we can remove them if you like